File: source/code/nodejs/account_management/update_dir_pwd_strength_req.js
directory.getPasswordPolicy(function (err, passwordPolicy) {
  if (err) {
    return console.error(err);
  }

  passwordPolicy.getStrength(function (err, passwordStrengthPolicy) {
    if (err) {
      return console.error(err);
    }

    passwordStrengthPolicy.minLength = 1;
    passwordStrengthPolicy.maxLength = 24;
    passwordStrengthPolicy.minSymbol = 1;

    passwordStrengthPolicy.save(function (err) {
      if (err) {
        return console.error(err);
      }

      console.log('Password strength policy updated!');
    });
  });
});

File: source/code/csharp/account_management/update_dir_pwd_strength_req.cs
// Get the Password Strength Policy from the Directory's Password Policy resource
var passwordPolicy = await myDirectory.GetPasswordPolicyAsync();
var strengthPolicy = await passwordPolicy.GetPasswordStrengthPolicyAsync();

// Update and save
strengthPolicy.SetMinimumLength(1)
    .SetMaximumLength(24)
    .SetMinimumSymbols(1);
await strengthPolicy.SaveAsync();
